price_parser/parser.py

```diff
@@ -180,6 +183,10 @@ def extract_price_text(price: str) -> Optional[str]:
     >>> extract_price_text("50")
     '50'
     """
+
+    if date_format(price):
```

Comment:
What if a string contains both?

For example:
```
30€ until May, 2005, 35€ afterwards
2019-08-19: 22 USD
105 EUR at July, 2004
$10 EUR during March, 2016
```

Reply:
@Gallaecio fixed in https://github.com/scrapinghub/price-parser/pull/19/commits/32ae1296c946c1e53bb6cf6ff68ecdb2b3f649c7